birdhouse/optional-components/wps-healthchecks/docker-compose-extra.yml

```diff
@@ -20,7 +20,7 @@ services:
           "CMD",
           "curl",
           "--fail",
-          "http://localhost:${HTTP_PORT}?service=WPS&request=GetCapabilities",
+          "http://localhost:5000?service=WPS&request=GetCapabilities",
```

Comment:
Just curious, in this commit cd8082aae5de3e365432fe59e7e33bbccbfdf342, `HTTP_PORT` did not work?

Reply:
No. I have tried also using `$$HTTP_PORT` so that it gets resolve only inside the container, but still didn't work.